app/services/notification_search.rb

```diff
@@ -12,21 +12,35 @@ def initialize(params, current_user)
   private
 
     def filters_def
-      [{ start_input_label: 'Start Date',
-         end_input_label: 'End Date',
-         column: :created_at,
-         type: RadCommon::DateFilter },
-       { column: 'notification_type_id', options: notification_type_options },
-       { column: 'content', type: RadCommon::LikeFilter },
-       { input_label: 'Record Type', column: 'record_type', options: record_type_options },
-       { input_label: 'Record ID', column: 'record_id', type: RadCommon::EqualsFilter, data_type: :integer }]
+      items = []
```

Comment:
shows/hides the user filter and sort if user is an admin, but defaults the filter to current user